daisi/src/cpps/agv/agv_logical.cpp

```diff
@@ -230,7 +230,7 @@ void AgvLogicalNs3::processMessageUpdate(const AmrStatusUpdate &payload) {
 }
 
 void AgvLogicalNs3::processOrderStatusUpdate(const AmrOrderUpdate &payload) {
-  task_management_->setCurrentOrderStatus(payload.getState());
+  task_management_->setCurrentOrderStatus(payload.getState());  // TODO rework
```

Comment:
I remember adding this TODO, doing the rework and then deleting it again because agv_logical would be deleted. Leaving the comment in was an oversight.